composer.json

```diff
@@ -9,7 +9,7 @@
         "drupal/webform": "^5.2"
     },
     "suggest": {
-        "dpc-sdp/tide_api:^1.2.6": "Allows to use Drupal in headless mode"
+        "dpc-sdp/tide_api:^1.2.9": "Set default value to jsonapi.settings readonly property"
```

Comment:
@anthony-malkoun @vincent-gao Incorrect description of tide_api.

Reply:
I got @vincent-gao to push a quick feature branch to fix this. When the build completes I can merge. @sonnykt are you creating the tag or do you need me to do it?